defaults/main.yml

```diff
@@ -220,3 +220,5 @@ bbb_firewall_ufw:
       - rule: allow
         port: 22
         proto: tcp
+
+bbb_docker_compose_version: 1.24.1
```

Comment:
Would it be possible to update docker-compose to the latest release?

Reply:
Sure, this is fixed in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/124/commits/8f220dee924d448b4d6f3f6d589b82e9ef6c53ed
Thank you for reviewing the PR @Davka